dev/ci/README.md

```diff
@@ -162,3 +162,26 @@ but if you wish to save more time you can skip the job by setting
 This means you will need to change its value when the Docker image
 needs to be updated. You can do so for a single pipeline by starting
 it through the web interface.
+
+Setting up runners
+------------------
+
+### Windows
+
+General documentation here:
+
+https://docs.gitlab.com/runner/install/windows.html
+
+Here are the steps to prepare specifically an Inria CloudStack VM to work as a runner for our Gitlab:
+
+- Download Gitlab Runner at
+  https://gitlab-runner-downloads.s3.amazonaws.com/latest/binaries/gitlab-runner-windows-amd64.exe
+  and save as C:\Gitlab-Runner\gitlab-runner.exe
+- Install git (64 bits) from https://git-scm.com/download/win, to C:\Program Files\git
+- When asked, choose autocrlf=input
+- Install 7zip from https://www.7-zip.org/download.html, to C:\Program Files\7-Zip
+- Start an administrator command prompt
+- Run gitlab-runner.exe register and give the relevant settings (runner should use the tag "windows")
+- Run gitlab-runner.exe install
+- Run gitlab-runner.exe start
+- Reboot the VMs (will put git in path for shell jobs)
```

Comment:
Well, you are critically missing the runner registration part. I think this should stay on its own section, and contain two subsections:
- Installation of the runner software [on windows, linux]
- Configuration of the runners with Coq's token [+ tags, etc]...

Reply:
Ok, but the second part is described in the general documentation I linked to. Not sure we want to keep a copy in sync.

Reply:
The link is good to me.